source/Coop/Mod/ObjectManagerAdapter.cs

```diff
@@ -226,14 +226,14 @@ static void Postfix(Clan __instance)
                 Instance.OnAfterRegisterObjectWithoutInitialization(__instance);
             }
         }
-        [HarmonyPatch(typeof(Hero), "OnBeforeLoad")]
-        private static class Hero_OnBeforeLoad
-        {
-            static void Postfix(Hero __instance)
-            {
-                Instance.OnAfterRegisterObjectWithoutInitialization(__instance);
-            }
-        }
+        //[HarmonyPatch(typeof(Hero), "OnBeforeLoad")] Not used, replacement?
```

Comment:
From a quick look they progressed a bit further with their init logic refactoring. The introduced a new attribute `[LoadInitializationCallback]`. All methods with that attribute are called called in `LoadResult.InitializeObjects`.  `MBObjectBase.OnBeforeLoad` is part of that initialization. So i guess the whole workaround for all these `OnBeforeLoad` is now redundant because they fixed it themselves.

With current development, this is unused anyways. So i think it's fine to just delete and have a look again if we need it.